flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/ExecutionContext.java

```diff
@@ -468,30 +496,7 @@ public ExecutionConfig getExecutionConfig() {
 			}
 		}
 
-		public JobGraph createJobGraph(String name) {
-			final Pipeline pipeline = createPipeline(name, flinkConfig);
-
-			int parallelism;
-			if (execEnv != null) {
-				parallelism = execEnv.getParallelism();
-			} else if (streamExecEnv != null) {
-				parallelism = streamExecEnv.getParallelism();
-			} else {
-				throw new RuntimeException("No execution environment defined.");
-			}
-			JobGraph jobGraph = FlinkPipelineTranslationUtil.getJobGraph(
-					pipeline,
-					flinkConfig,
-					parallelism);
-
-			jobGraph.addJars(executionParameters.getJars());
-			jobGraph.setClasspaths(executionParameters.getClasspaths());
-			jobGraph.setSavepointRestoreSettings(executionParameters.getSavepointRestoreSettings());
-
-			return jobGraph;
-		}
-
-		private Pipeline createPipeline(String name, Configuration flinkConfig) {
+		public Pipeline createPipeline(String name, Configuration flinkConfig) {
```

Comment:
In this method it seems we do not use the `flinkConfig` or the `parallelism` in  the `else` branch. Is it ok to remove them? Also the `else` is not needed as we `return` in the `if` branch. So this can become: 
```
public Pipeline createPipeline(String name) {
			if (streamExecEnv != null) {
				// special case for Blink planner to apply batch optimizations
				// note: it also modifies the ExecutionConfig!
				if (executor instanceof ExecutorBase) {
					return ((ExecutorBase) executor).getStreamGraph(name);
				}
				return streamExecEnv.getStreamGraph(name);
			}
			return execEnv.createProgramPlan(name);
		}
```